opdract1.js

```diff
@@ -0,0 +1,49 @@
+//pseudo-code
+
+//1a:
+//
+//
+
+
+
+const tvNames = inventory.map ((tv) => {
+    return tv.type
+})
+console.log(tvNames)
+
+//1b:
+// 1. gebruik de filter-methods
+// 2. Filter uitkomst
+
+const tvSoldOut = inventory.filter((tvSoldOutActual) => {
+    return tvSoldOutActual.originalStock === tvSoldOutActual.sold
+})
+
+console.log(tvSoldOut)
+
+//1c:
+
+// 1. gebruik de filter methode
+// 2. Via options van het object options om bij ambilight te komen
+// 3. Filter uitkomst
+
+const tvAmbiLight =  inventory.filter((tvAmbi) => {
+    return tvAmbi.options.ambiLight === true
+})
```

Comment:
Omdat `ambiLigth` van zichzelf al een boolean is hoef je er niet expliciet een check aan toe te voegen:
```suggestion
const tvAmbiLight =  inventory.filter(tvAmbi => tvAmbi.options.ambiLight);
```